webpack.config.js

```diff
@@ -75,6 +75,30 @@ const babelLoader = {
 	},
 };
 
+/**
+ * Converts @wordpress require into window reference
+ *
+ * Note this isn't the same as camelCase becuase of the
+ * way that numbers don't trigger the capitalized next letter
+ *
+ * @example
+ * wordpressRequire( '@wordpress/api-fetch' ) = 'wp.apiFetch'
+ * wordpressRequire( '@wordpress/i18n' ) = 'wp.i18n'
+ *
+ * @param {string} request import name
+ * @return {string} global variable reference for import
+ */
+const wordpressRequire = request => {
+	const [ , /* @wordpress */ name ] = request.split( '/' );
+
+	return `wp.${ name.replace( /-([a-z])/, ( match, letter ) => letter.toUpperCase() ) }`;
```

Comment:
I see that you've removed the global flag `/g`, but this will not work as expected for imports that contain more than one dash in the second part of the name. In other words `@wordpress/block-serialization-spec-parser` will become `wp.blockSerialization-spec-parser`. We should bring back the global flag.

Reply:
oops!

Reply:
fixed!